_layouts/staff.html

```diff
@@ -0,0 +1,44 @@
+---
+layout: default
+---
+
+<div class="staff">
```

Comment:
don't need this div, the [new `page` layout](https://github.com/illinois-cs498vr/illinois-cs498vr.github.io/blob/master/_layouts/page.html#L12) has a div that wraps the content of child layouts with a classname of the child layout name.